docs/src/main/asciidoc/security-identity-providers.adoc

```diff
@@ -17,7 +17,12 @@ In the Quarkus Security framework, identity providers play a crucial role in aut
 [[identity-providers]]
 `IdentityProvider` converts the authentication credentials provided by `HttpAuthenticationMechanism` to a `SecurityIdentity` instance.
 
-Some extensions, for example, `OIDC`, `OAuth2`, and `SmallRye JWT`, have inline `IdentityProvider` implementations specific to the supported authentication flow.
+ifndef::no-quarkus-elytron-security-oauth2[]
```

Comment:
I propose to simplify by dropping a ref to OAuth2, since this sentence gives an example only

Reply:
Updated. Thanks.